Sources/Subs-Compat.php

```diff
@@ -24,28 +33,4918 @@
  * SMF\Config::$backward_compatibility support
  *********************************************/
 
+
 if (!empty(SMF\Config::$backward_compatibility)) {
 	/*
 	 * In SMF 2.x, there was a file named Subs.php that was always loaded early in
 	 * the startup process and that contained many utility functions. Everything
 	 * else assumed that those functions were available. Subs.php went the way of
 	 * the dodo in SMF 3.0 after all its functions were migrated elsewhere, but
 	 * mods that rely on backward compatibilty support will still expect all those
-	 * functions to be available. So if backward compatibilty support is enabled,
-	 * we need to load a bunch of classes in order to make them available.
-	 */
-	class_exists('SMF\\Attachment');
-	class_exists('SMF\\BBCodeParser');
-	class_exists('SMF\\Logging');
-	class_exists('SMF\\PageIndex');
-	class_exists('SMF\\Theme');
-	class_exists('SMF\\Time');
-	class_exists('SMF\\TimeZone');
-	class_exists('SMF\\Topic');
-	class_exists('SMF\\Url');
-	class_exists('SMF\\User');
-	class_exists('SMF\\Graphics\\Image');
-	class_exists('SMF\\WebFetch\\WebFetchApi');
+	 * functions to be available.
+	 */
+
+	function Activate()
+	{
+		return Actions\Activate::call();
+	}
```

Comment:
Here and below, there is a lack of indentation between functions

Reply:
Fixed